app/admin.py

```diff
@@ -404,6 +409,27 @@ def has_delete_permission(self, request, obj=None, *args, **kwargs):
         return False
 
 
+class DownloadableUserAdmin (UserAdmin):
+    actions = ['download_csv']
+
+    def download_csv(self, request, queryset):
+        f = StringIO.StringIO()
+        writer = csv.writer(f)
+        writer.writerow(["email", "mobile_number", "display_name", "gender",
+                        "year_of_birth", "identity"])
+        for user in queryset:
+            user_profile = user.userprofile_set.all()[0]
+
+            writer.writerow([user.email, user_profile.mobile_number,
+                            user_profile.alias, user_profile.gender,
+                            user_profile.year_of_birth, user_profile.identity])
+
+        f.seek(0)
+        response = HttpResponse(f, content_type='text/csv')
```

Comment:
It may be easier to do `f.getvalue()` here instead of the `f.seek(0)` and giving the `StringIO` object to the `HttpResponse` but this is bikeshedding since the code works and you have tests to prove it :) :cake: